test/utils/data.js

```diff
@@ -752,7 +752,66 @@ export function project2000 () {
     }
   ]))
   .then(() => putObjectFromFile(bucket, 'scenario-2000/road-network_000000', FILE_ROAD_NETWORK))
-  .then(() => putObjectFromFile(bucket, 'scenario-2000/poi_000000', FILE_POI));
+  .then(() => putObjectFromFile(bucket, 'scenario-2000/poi_000000', FILE_POI))
+  .then(() => scenarioResults(
+    [
+      {
+        'id': 1,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200001,
+        'project_aa_id': 200001
+      },
+      {
+        'id': 2,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200002,
+        'project_aa_id': 200001
+      },
+      {
+        'id': 3,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200003,
+        'project_aa_id': 200001
+      }
+    ]
+  ))
+  .then(() => scenarioResultsPOI(
```

Comment:
@danielfdsilva 
This is how the results are written to the db right now:
![image](https://user-images.githubusercontent.com/751330/27393182-93bedff4-5676-11e7-882e-e63a03eda9d9.png)